framework/dependencies.yml

```diff
@@ -34,47 +34,46 @@ require:    &allDependencies
     - javax.inject 1.0
     - javax.validation -> validation-api 1.1.0.Final
     - jaxen 1.2.0
-    - joda-time 2.10
+    - joda-time 2.10.13
     - org.hamcrest -> hamcrest-all 1.3
-    - junit 4.12
+    - junit 4.13.2
     - jregex 1.2_01
-    - log4j 1.2.17
-    - net.sf.ehcache -> ehcache 2.10.6
+    - net.sf.ehcache -> ehcache 2.10.9.2
     - net.sf.ezmorph -> ezmorph 1.0.6
     - net.sf.jsr107cache -> jsr107cache 1.1
-    - net.sf.oval -> oval 1.86
-    - mysql -> mysql-connector-java 8.0.22
+    - net.sf.oval -> oval 1.90
+    - mysql -> mysql-connector-java 8.0.28
     - oauth.signpost -> signpost-core 1.2.1.2
     - org.apache.geronimo.specs -> geronimo-servlet_2.5_spec 1.2
-    - org.apache.ivy -> ivy 2.4.0
-    - org.bouncycastle -> bcprov-jdk15on 1.61
-    - org.bouncycastle -> bcpkix-jdk15on 1.61
-    - org.codehaus.groovy -> groovy 3.0.6
-    - org.codehaus.groovy -> groovy-xml 3.0.6
-    - org.eclipse.jdt -> org.eclipse.jdt.core 3.23.0
+    - org.apache.ivy -> ivy 2.5.0
+    - org.bouncycastle -> bcprov-jdk15on 1.70
+    - org.bouncycastle -> bcpkix-jdk15on 1.70
+    - org.codehaus.groovy -> groovy 3.0.9
+    - org.codehaus.groovy -> groovy-xml 3.0.9
+    - org.eclipse.jdt -> org.eclipse.jdt.core 3.28.0
     - org.hibernate -> hibernate-core 5.4.30.Final.patched
-    - net.bytebuddy -> byte-buddy 1.10.21
+    - net.bytebuddy -> byte-buddy 1.12.7
     - javax.persistence -> javax.persistence-api 2.2
     - org.hibernate.common -> hibernate-commons-annotations 5.1.2.Final
     - org.hibernate -> hibernate-validator 5.4.1.Final
-    - org.jboss -> jandex 2.2.3.Final
-    - org.jboss.logging -> jboss-logging 3.4.1.Final
+    - org.jboss -> jandex 2.4.2.Final
+    - org.jboss.logging -> jboss-logging 3.4.3.Final
     - org.jboss.spec.javax.transaction -> jboss-transaction-api_1.2_spec 1.1.1.Final
-    - javax.persistence -> javax.persistence-api 2.2 
     - javax.xml.bind -> jaxb-api 2.3.1
-    - org.glassfish.jaxb -> jaxb-runtime 2.3.1
+    - org.glassfish.jaxb -> jaxb-runtime 2.3.5
     - com.fasterxml -> classmate 1.5.1
     - org.hibernate -> hibernate-c3p0 5.4.30.Final
     - org.hibernate -> hibernate-ehcache 5.4.30.Final
-    - com.mchange -> mchange-commons-java 0.2.12
-    - org.javassist -> javassist 3.27.0-GA
+    - com.mchange -> mchange-commons-java 0.2.20
+    - org.javassist -> javassist 3.28.0-GA
     - io.netty -> netty 3.10.6.Final
-    - org.postgresql -> postgresql 42.2.18
-    - org.slf4j -> slf4j-api 1.7.22
-    - org.slf4j -> slf4j-log4j12 1.7.22
-    - org.yaml -> snakeyaml 1.17
-    - net.spy -> spymemcached 2.12.1
-    - com.thoughtworks.xstream -> xstream 1.4.9
+    - org.postgresql -> postgresql 42.3.1
+    - ch.qos.reload4j -> reload4j 1.2.18.3
```

Comment:
Can we put the change of log library in another pull request ?

Reply:
@xael-fry yes I'll revert that commit now

Reply:
Sorry I was thinking it is a pull request on play1, but it is only on your repo.

Reply:
@xael-fry yes right. I'm happy to revert the commit for the logging and then create a pull request for this branch to play1. Wdyt?

Reply:
yes the MR is ok for me, let's go

Reply:
I'm just resyncing my repo after the PRs you've just released

Reply:
The replace of Log4j with reload4j has been reverted